app/controllers/users_controller.rb

```diff
@@ -0,0 +1,62 @@
+#rubocop:disable all
```

Comment:
- [_**OPTIONAL**_] I think it would be helpful if you enable rubocop, since it helps you pick formatting errors and follow ruby style guide used by the community, which makes your code base easy to understand.